packages/design-system/src/components/OcTile/OcTile.vue

```diff
@@ -31,16 +35,14 @@
     </oc-resource-link>
     <div class="oc-card-body oc-p-s">
       <div class="oc-flex oc-flex-between oc-flex-middle">
-        <div class="oc-flex oc-flex-middle oc-text-truncate">
-          <oc-resource
-            :resource="resource"
-            :folder-link="resourceRoute"
-            @click="$emit('click', $event)"
-          />
+        <div class="oc-flex oc-flex-middle oc-text-truncate resource-name-wrapper">
+          <oc-resource :resource="resource" :folder-link="resourceRoute" @click="$emit('click')" />
```

Comment:
```suggestion
          <oc-resource :resource="resource" :folder-link="resourceRoute" :is-thumbnail-displayed="false" @click="$emit('click')" />
```

Thumbnails must be disabled here. Otherwise you might have glitchy situations where the OcTile has a large preview at the top and a small thumbnail in the bottom area. Managed to create that situation, don't have a formula to reproduce it, sorry. 👀